hypervisor/bsp/uefi/efi/Makefile

```diff
@@ -43,8 +43,8 @@ HOST  = $(shell $(CC) -dumpmachine | sed "s/\(-\).*$$//")
 ARCH := $(shell $(CC) -dumpmachine | sed "s/\(-\).*$$//")
 
 ifeq ($(ARCH),x86_64)
-	LIBDIR := $(shell if [ -d /usr/lib64 ]; then echo /usr/lib64; \
-			else if [ -d /usr/lib ]; then echo /usr/lib; fi ; fi;)
+	LIBDIR := $(shell if [ -e /usr/lib64/elf_x86_64_efi.lds ]; then echo /usr/lib64; \
```

Comment:
This is broken on Fedora as this file is put in a `gnuefi/` subdir (i.e. under `/usr/lib64/gnuefi/`).

Reply:
Thanks Geoff!  If @zhijianli88 can split this 2 patches to 2 PRs, I can merge the 1st one now.

Reply:
at my ubuntu16.04, gnu-efi is installed into /usr/lib instead of /usr/lib64
```
~/workspace$ dpkg -L gnu-efi | grep elf_x86_64_efi.lds
/usr/lib/elf_x86_64_efi.lds
```
but the previous Makefile try to link /usr/lib64/elf_x86_64_efi.lds becuase my host has /usr/lib64 as well which is provided by libc6-amd64:i386
```
~/workspace$ dpkg -S /usr/lib64
libc6-amd64:i386: /usr/lib64
```

the previous logic always assume LIBDIR is /usr/lib64 if it exists

Reply:
@gvancuts you are right, it might  break Fedora. let's consider more...